configure

```diff
@@ -1050,7 +1050,7 @@ def configure_intl(o):
   if not icu_ver_major:
     print ' Could not read U_ICU_VERSION_SHORT version from %s' % uvernum_h
     sys.exit(1)
-  icu_endianness = sys.byteorder[0];  # TODO(srl295): EBCDIC should be 'e'
+  icu_endianness = sys.byteorder[0];
```

Comment:
Please undo this change, or move it to a separate commit.  Rule of thumb: one change per commit, no more, no less.